src/pages/CSSocietyPage.js

```diff
@@ -116,8 +116,9 @@ export default function CSSocietyPage(props) {
               </Grid>
             ))}
           </Grid>
-        </Paper>
-        <AlumniAccordions members={alumni.compsoc} color="rgb(80 161 99)" />
+        </Paper> */}
+         <SocietyExec sid = {ecats.compsoc}/>
+        <AlumniAccordions members={alumni.compsoc} sid = {ecats.compsoc} color="rgb(80 161 99)" />
```

Comment:
why are we still passing the members as a prop, remove it

```suggestion
        <AlumniAccordions sid = {ecats.compsoc} color="rgb(80 161 99)" />
```